compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1718,6 +1731,19 @@ impl<'a> Parser<'a> {
                     self.struct_span_err(span, &msg)
                         .help("valid widths are 8, 16, 32, 64 and 128")
                         .emit();
+                } else if let Some(fixed) = fix_base_capitalisation(suf) {
+                    let msg = format!("invalid suffix `{}` for number literal", suf);
```

Comment:
```suggestion
                    let msg = format!("invalid prefix `{}` for number literal", suf);
```

Reply:
We might want this to be ``invalid prefix `0{}` for number`` I guess depending on how the error message looks.